spec/mailers/user_mailer_spec.rb

```diff
@@ -32,4 +32,18 @@
       @mail.body.encoded.should match("http://localhost:3000/g/#{@group.key}")
     end
   end
+
+  context 'sending email on being added to group' do
+    before :each do
+      @user = create(:user)
+      @inviter = create(:user)
+      @group = build(:group, full_name: "Group full name")
+      @mail = UserMailer.added_to_group(user: @user, inviter: @inviter, group: @group)
+    end
+
+    it 'renders the subject' do
+      @mail.subject.should == "#{@inviter.name} has added you to #{@group.full_name}"
```

Comment:
Hey @washburnad, would you mind swapping this to the newer (and less deprecated)

```
expect(@mail.subject).to eq "..."
```

Thanks!